septentrion/versions.py

```diff
@@ -1,24 +1,22 @@
+import dataclasses
+
 from septentrion import exceptions
 
 
+@dataclasses.dataclass(order=True, unsafe_hash=True, frozen=True)
 class Version:
-    def __init__(self, version_string: str):
+
+    version_tuple: tuple = dataclasses.field(compare=True)
+    original_string: str = dataclasses.field(compare=False)
```

Comment:
both could be comparable

Reply:
For this point I'm not sure. It could be pertinent but i prefer not do it now.